flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -1262,16 +1252,6 @@ protected abstract void internalDeregisterApplication(
             ApplicationStatus finalStatus, @Nullable String optionalDiagnostics)
             throws ResourceManagerException;
 
-    /**
-     * Allocates a resource using the worker resource specification.
-     *
-     * @param workerResourceSpec workerResourceSpec specifies the size of the to be allocated
-     *     resource
-     * @return whether the resource can be allocated
-     */
-    @VisibleForTesting
-    public abstract boolean startNewWorker(WorkerResourceSpec workerResourceSpec);
```

Comment:
Shall we also move `stopWorker` to `ActiveResourceManager`? `StandaloneResourceManager#stopWorker` does nothing.

Reply:
And maybe `workerStarted` as well. The method is used for deciding whether to accept a TM registration. For active clusters we accept only the resources that we actively requested, while for standalone clusters we always accept.